imp_services/logging.js

```diff
@@ -0,0 +1,68 @@
+exports.LOGTYPES = {
+    ADD               : {value: 100, name: "Add"},
+    REMOVE            : {value: 200, name: "Remove"},
+    AUDIT             : {value: 300, name: "Audit"},
+    NOTE              : {value: 400, name: "Note"},
+    NEWPRODUCTCREATED : {value: 500, name: "New Product Created"}
+};
+
+/* Use of Logs.GenericVar in DB:
+ ADD - quantity added
+ REMOVE - quantity removed
+ AUDIT - unused
+ NOTE - unused
+ NEWPRODUCTCREATED - unused
+ */
+
+/* This method is used for updating the log, and can be nested within a .then() promise statement.
```

Comment:
Really great documentation here, but if we are going to allow .then nesting, we should make sure that this function implements a q.defer. Its ok to add this as a TODO for now.